### ISSUE TITLE
Update app website

### ISSUE THREAD

**Issue by navingator** [Beamanator/Ulti-Mate#50]
We currently don't own the domain scoober.io
We should acquire it or change the domain name for release.

**Comment by Beamanator** [comment]
domain search in progress.

Possible list for needs in website:
- [ ] Link to Google play store (after app is published)
- [ ] Name of App / group
- [ ] Slogan
- [ ] About us page (contact)
- [ ] Tutorial for using app
- [ ] Highlight features of app
- [ ] Upcoming app updates & tentative dates

Future:
- [ ] Database integration

**Comment by navingator** [comment]
Agreed, though it may make more sense for the website to highlight features of our app instead of a tutorial.

**Comment by Beamanator** [comment]
Nice. Added. I'm going to start with just the title. Probably on a new github repo. Also, I'm thinking of using a free template just to get things going quicker. Thoughts? We'd have to include their copyright at the bottom, but we'd at least have all the code. Not going to use anything like wordpress

**Comment by Beamanator** [comment]
If we use freewebsitetemplates.com, we only have to leave references to the site in commented html / css code so it doesn't actually show on the page. Not bad.

**Comment by Beamanator** [comment]
For the highlights / features page - I'm thinking it would be cool to have like 6 - 10 screenshots of activities in the app [listed below] and when a user hovers the mouse over each picture, some text would pop up describing what's cool about that feature.

Possible features:
- [ ] Templates
- [ ] Hard / Soft Caps
- [ ] Sharing
- [ ] Notifications [persistent & hard / soft caps]
- [ ] Old game tracking
- [ ] Home page [live game]
- [ ] others?
